src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx

```diff
@@ -0,0 +1,148 @@
+import currency from "currency.js"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { toSentence } from "underscore.string"
+
+import { SeoProductsForArtworks_artworks } from "__generated__/SeoProductsForArtworks_artworks.graphql"
+import { Product } from "./Product"
+
+const { APP_URL } = process.env
+
+const AVAILABILITY = {
+  "for sale": "https://schema.org/InStock",
+  sold: "https://schema.org/OutOfStock",
+}
+
+const formatCurrency = value => currency(value, { separator: "" }).format()
+
+interface SeoProductsProps {
+  artworks: SeoProductsForArtworks_artworks
+}
+
+export class SeoProducts extends React.Component<SeoProductsProps> {
+  render() {
+    const {
+      artworks: { artworks_connection },
+    } = this.props
+
+    // here the filtering is necessary so we can re-use the artwork list shown in the page (could include
+    // non-acquireable artworks) without making an extra request.
+    const artworksForSeoProduct = artworks_connection.edges.filter(
+      edge => edge.node.is_acquireable && edge.node.partner
+    )
+
+    return artworksForSeoProduct.map(({ node }, index) => {
+      const { artists, is_price_range, partner, price } = node
+      const location = partner.locations && partner.locations[0]
+      const artistsName = artists
+        ? toSentence(node.artists.map(({ name }) => name))
+        : null
+      const isInstitution = partner.type === "Institution"
+
+      return (
+        <Product
+          key={node.__id}
+          data={{
+            name: node.title,
+            image: node.image.url,
+            description: node.meta.description,
```

Comment:
Probably worth checking if `node.image` and `node.meta` exist, we came upon some bad errors when deep objects came up undefined when implementing this for artist pages.

Reply:
Could you send me some artwork examples that don't have `image` or `meta` so I can test this?